tox.ini

```diff
@@ -5,7 +5,7 @@ toxworkdir={env:TOX_WORK_DIR:.tox}
 deps =
 	# Ideally all the dependencies should be set as "extras"
 	# workaround for pytest-dev/execnet#195
-	execnet @ git+https://github.com/jaraco/execnet@bugfix/195-encodingwarning
```

Comment:
I tried to run `tox -- --mypy` locally and it was failing because this branch no longer exists. That change was included in `2.0.0` according to the changelog.